src/index.js

```diff
@@ -1,10 +1,7 @@
 /* ___________ REQUIRE CORE DEPENDENCIES AND CONFIG FILES ___________ */
 
-const Discord = require("discord.js");
-require("dotenv").config();
-require("./database");
-import "core-js/stable";
-import "regenerator-runtime/runtime";
+import {client} from './core';
+import 'dotenv/config';
```

Comment:
This is awesome! ✨ Just a brief reminder for us to require the `dotenv` first before importing `client` and remove the additional `dotenv` configs from the `src/core` modules like we discussed earlier!